modules/modifying-an-existing-ingress-controller.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-an-existing-ingress-controller_{context}"]
+= Modifying an existing Ingress Controller
+
+As a cluster administrator, you can modify an existing Ingress Controller to manage the DNS record lifecycle.
```

Comment:
```suggestion
As a cluster administrator, you can modify an existing Ingress Controller to manually manage the DNS record lifecycle.
```

Reply:
++1 to adding the term "manual" . Thanks .